OpenRA.Game/FileFormats/Png.cs

```diff
@@ -0,0 +1,211 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net;
+using System.Text;
+
+namespace OpenRA.FileFormats
+{
+	public class Png
```

Comment:
Can you please move this to `OpenRA.Mods.Common.FileFormats`? This doesn't appear to be used anywhere in `OpenRA.Game`.

Reply:
There seems to be something weird going on with the encoding on this file - the indentation displayed by GitHub is inconsistent with the normal formatting.

Reply:
I was thinking about that too, but i thought png is a so standard format, that it would almost be an engine specific feature, while common ist mostly the rts core module. But i dont mind, sure i can move it.